src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -88,7 +95,7 @@ define 'orifice-calculator-viewmodel', ['knockout', 'lodash'], (ko, _) ->
         _.ceil (1 / Math.sqrt(1 - @betaRatio() ** 4)), 2
 
       @flowRate = ko.computed =>
-        operatingTemperatureInRankine = +@operatingTemperature() + ABSOLUTE_ZERO
+        operatingTemperatureInRankine = @operatingTemperature() + ABSOLUTE_ZERO
 
         flowRate = UNIT_CONVERSION_FACTOR * COEFFICIENT_OF_DISCHARGE * EXPANSION_FACTOR * @velocityOfApproach() * @orificeBoreDiameter() ** 2 * BASE_TEMPERATURE/BASE_PRESSURE \
```

Comment:
You don't need to escape coffeescript to get new line.
Please make it more readable:

```

        flowRate = UNIT_CONVERSION_FACTOR * COEFFICIENT_OF_DISCHARGE * EXPANSION_FACTOR *
          @velocityOfApproach() * @orificeBoreDiameter() ** 2 * BASE_TEMPERATURE / BASE_PRESSURE *
          ((@operatingPressure() * BASE_COMPRESSIBILITY * @differentialPressure()) /
          (@baseSpecificGravity() * @compressibilityCorrectionValue() * operatingTemperatureInRankine)) ** 0.5 / 60
```